r4/v2-to-fhir-custom-mapper-service/service.bal

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
```

Comment:
Move to a sub folder `data-mappers`.

Reply:
fixed with 4ce07f9f38de973d0024c750e49923240e35fc3d